be/src/exec/tablet_sink.cpp

```diff
@@ -503,6 +516,8 @@ void NodeChannel::try_send_batch(RuntimeState* state) {
 
     // tablet_ids has already set when add row
     request.set_packet_seq(_next_packet_seq);
+    LOG(INFO) << "try to send " << row_batch->num_rows() << " row to " << this->_node_id
```

Comment:
That will print too much logs

Reply:
Already delete this line along with a lot of useless log lines